monkey/infection_monkey/network/relay/sockets_pipe.py

```diff
@@ -22,13 +23,15 @@ def __init__(
         source,
         dest,
         pipe_closed: Callable[[SocketsPipe], None],
+        pipe_received_data: Callable[[socket.socket, bytes], None],
```

Comment:
Should we prefix these with "on_", (e.g. "on_pipe_closed()"). That's a fairly common naming convention for callbacks.

Reply:
Yeah, I was following the existing precedent here, but an "on_" prefix makes more sense

Reply:
I'm suggesting we could fix them all.

Reply:
All under `network/relay` you mean? From what I can tell, that would be:
- `TCPConnectionHandler`'s `client_connected` and
- `TCPPipeSpawner`'s `handle_pipe_data`

Reply:
Yeah, your call.